sklearn/feature_extraction/tests/test_text.py

```diff
@@ -1284,10 +1285,13 @@ def test_tfidf_transformer_type(X_dtype):
     assert X_trans.dtype == X.dtype
 
 
-def test_tfidf_transformer_sparse():
+@pytest.mark.parametrize(
+    "csc_container,csr_container", zip(CSC_CONTAINERS, CSR_CONTAINERS)
```

Comment:
you can change this to "sparse_container", CSC_CONTAINERS + CSR_CONTAINERS

Reply:
Nop, we want to make a combination of CSR and CSC on the other side. We should make the product to be sure that whatever combination, it will provide the same results.